.env.local.example

```diff
@@ -1,5 +1,5 @@
 NEXTAUTH_URL=http://localhost:3000
-NEXTAUTH_SECRET​= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
+SECRET= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
```

Comment:
This is not an error, it is a new environment variable we introduced 💭
https://next-auth.js.org/configuration/options#nextauth_secret

Reply:
ok, than change it please in https://github.com/nextauthjs/next-auth-example/blob/main/pages/api/examples/jwt.ts

Reply:
I changed it.